README.md

```diff
@@ -15,7 +15,7 @@ CLI tool to generate github contributors hall of fame for github readme
 ## Install
 
 ```shell
-$ npm i -g rhof-cli
+$npm -g i rhof
```

Comment:
let's also add a space after `$` to make it clear, that $ is for command line. Its not part of the install command